gov2/kms/README.md

```diff
@@ -0,0 +1,96 @@
+# AWS SDK for Go V2 code examples for AWS AWS KMS 
+
+## Purpose
+
+These examples demonstrates how to perform several AWS Key Management Service (AWS KMS) 
```

Comment:
These examples demonstrate how
(remove the 's' from demonstrate)

Reply:
Fixed